src/installer/pkg/packaging/rpm/package.targets

```
@@ -315,84 +318,31 @@
-    <MakeDir Directories="$(RpmLayoutDocs)" />
-    <MakeDir Directories="$(RpmLayoutTemplates)" />
-    
-     <!-- Copy files to rpm layout -->
-    <ItemGroup>
-         <RDTemplatesFiles Include="$(TemplatesDir)/**/*" />
-    </ItemGroup>
-
-    <Copy SourceFiles="@(RDTemplatesFiles)" DestinationFiles="@(RDTemplatesFiles->'$(RpmLayoutTemplates)/%(RecursiveDir)%(Filename)%(Extension)')" />
-
-    <!-- Replace config json variables -->
-    <ItemGroup>
-      <RuntimeDependenciesTokenValue Include="%RUNTIME_DEPS_RPM_PACKAGE_NAME%">
-        <ReplacementString>$(RuntimeDependenciesRpmPkgName)</ReplacementString>
-      </RuntimeDependenciesTokenValue>
-      <RuntimeDependenciesTokenValue Include="%RUNTIME_DEPS_VERSION%">
-        <ReplacementString>$(RuntimeDepsRpmPackageVersion)</ReplacementString> 
-      </RuntimeDependenciesTokenValue>
-      <RuntimeDependenciesTokenValue Include="%RUNTIME_DEPS_REVISION%">
-        <ReplacementString>$(RuntimeDepsRpmPackageRelease)</ReplacementString>
-      </RuntimeDependenciesTokenValue>
+    <!-- non-portable build - add OutputRid to new item list -->
+    <ItemGroup Condition="'$(PortableBuild)' == 'false'">
+      <DepsProjects Include="$(rpmPackagingConfigPath)pack-deps.proj">
+        <Properties>DepsRid=$(OutputRid);$(SharedProperties)</Properties>
+      </DepsProjects>
     </ItemGroup>
-    
-    <ReplaceFileContents InputFile="$(ConfigJsonFile)"
-                         DestinationFile="$(RpmLayoutDirectory)$(rpmConfigJsonName)"
-                         ReplacementItems="@(RuntimeDependenciesTokenValue)" />
 
-    <!-- Call the task to build the pre-reqs (parameters, copyright, changelog) for calling the FPM tool -->
-    <BuildFPMToolPreReqs  InputDir="$(RpmLayoutDirectory)"
-                          OutputDir="$(RpmIntermediatesDir)"
-                          PackageVersion="$(RpmPackageVersion)"
-                          ConfigJsonFile="$(RpmLayoutDirectory)$(rpmConfigJsonName)">
-                          <Output TaskParameter="FPMParameters" PropertyName="FPMCmdParameters" />
-    </BuildFPMToolPreReqs>
-
-    <!-- Build the RPM package by calling the FPM tool and passing the parameter list -->
-    <Exec Command="fpm $(FPMCmdParameters)"  WorkingDirectory="$(RpmIntermediatesDir)" />
-
-    <!-- Copy package to output -->
-    <ItemGroup>
-      <GeneratedRpmFiles Remove="@(GeneratedRpmFiles)" />
-      <GeneratedRpmFiles Include="$(RpmIntermediatesDir)/*.rpm" />
+    <!-- portable build - find all config json deps files and add their OutputRid values to new item list -->
+    <!-- iterate thru all items in item list and call pack-deps project -->
+    <ItemGroup Condition="'$(PortableBuild)' == 'true'">
+      <ConfigJsonFilenames Include="$(rpmPackagingConfigPath)$(DepsConfigFilePrefix)*.json" />
+      <Rids Include="$([System.Text.RegularExpressions.Regex]::Match(%(ConfigJsonFilenames.Filename),'$(DepsConfigFilePrefix)([A-Za-z0-9_\-\.]+)').Groups[1].Value)" />
```

A little bit of formatting--quotes and space after `,`.

```suggestion
      <Rids Include="$([System.Text.RegularExpressions.Regex]::Match('%(ConfigJsonFilenames.Filename)', '$(DepsConfigFilePrefix)([A-Za-z0-9_\-\.]+)').Groups[1].Value)" />
```

---

Will fix.